Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -473,17 +483,27 @@ bool PPCSymbolDB::SaveSymbolMap(const std::string& filename) const
   for (const auto& symbol : function_symbols)
   {
     // Write symbol address, size, virtual address, alignment, name
-    f.WriteString(fmt::format("{0:08x} {1:08x} {2:08x} {3} {4}\n", symbol->address, symbol->size,
-                              symbol->address, 0, symbol->name));
+    std::string line = fmt::format("  {0:08x} {1:06x} {2:08x} {3} {4}", symbol->address,
+                                   symbol->size, symbol->address, 0, symbol->name);
```

Comment:
I'm pretty sure these leading spaces weren't there last time I reviewed this part. They should be removed.

Reply:
Still not addressed.